client/transactions/list/test/index.tsx

```diff
@@ -415,14 +426,6 @@ describe( 'Transactions list', () => {
 			} );
 		} );
 
-		afterEach( () => {
-			jest.resetAllMocks();
-		} );
```

Comment:
Similar to my comment above, this was causing issues with the `@wordpress/data` mock, and it seemed as though it might not be needed for the CSV tests, so I removed them altogether. Please let me know if they actually are needed for some reason and perhaps we can explicitly reset/clear the relevant mocks rather than all.

Reply:
>this was causing issues with the @wordpress/data mock

Were you facing the similar error as above when these lines were present?

Reply:
Yup. Specifically the following error:

```
TypeError: Cannot read property 'createNotice' of undefined
```

I suspect resetting/restoring all mocks is undoing the `@wordpress/data` mock hack implemented above. But I'm not entirely sure tbh. Any thoughts on this?